include/open62541/server.h

```diff
@@ -1892,11 +1892,10 @@ UA_Server_setExpirationDate(UA_Server *server, const UA_NodeId conditionId,
 
 UA_StatusCode UA_EXPORT
 UA_Server_updateCertificate(UA_Server *server,
-                            const UA_ByteString *oldCertificate,
-                            const UA_ByteString *newCertificate,
-                            const UA_ByteString *newPrivateKey,
-                            UA_Boolean closeSessions,
-                            UA_Boolean closeSecureChannels);
+                            const UA_NodeId certificateGroupId,
+                            const UA_NodeId certificateTypeId,
+                            const UA_ByteString certificate,
+                            const UA_ByteString *privateKey);
```

Comment:
This needs documentation.
Why is the privateKey a pointer and the others are not?
Can we have privateKey == NULL if we don't have it?

Otherwise make it a non-pointer argument also.

Reply:
I will update the documentation. If no private key is specified, the system will attempt to use either the existing key or the one generated during the CSR creation process.